sticker-graph/sticker_graph/rnn_model.py

```diff
@@ -22,6 +22,79 @@ def dropout_wrapper(
         state_keep_prob=state_keep_prob)
 
 
+def stack_bidirectional_dynamic_rnn(cells_fw,
```

Comment:
We should put a comment up here with the source of the original function. It would be nicer if it was in a separate file (maybe `vendored.py` ?), because then we could also include the copyright header from:

https://github.com/tensorflow/tensorflow/blob/r1.13/tensorflow/contrib/rnn/python/ops/rnn.py

It should probably also have a 1-2 line description of what was modified compared to the upstream function.

Reply:
The same thing should also happen in `write_helper.py` for `_create_file_writer_generic_type`. Both things in one go, or `stack_bidirectional_rnn` now and then in another PR `_create_file_writer_generic_type`?

Reply:
The `_create_file_writer_generic_type` change should go in a separate PR.